### ISSUE TITLE
SetSignature gets styles overwritten when a normalemail.dotm file is present in Outlook Desktop (classic)

### ISSUE THREAD

**Issue by bogdanst24** [OfficeDev/office-js#4929]
## Your Environment

* Platform [PC desktop, Mac, iOS, Office on the web]: PC Desktop
* Host [Excel, Word, PowerPoint, etc.]: Outlook (classic/legacy)
* Office version number: Any
* Operating System: Windows
 
## Expected behavior

The result is consistent regardless of the Platform when using the setSignature API with any HTML.

## Current behavior
When having a template set for new documents (emails) in Outlook in the form of a custom normalemail.dotm file, then the signature inserted will inherit the document's styles. It is not the same when:
- using Web Add-ins in Outlook Online, even though a custom style is set for any new emails
- using Outlook's native signatures when having a custom template for new emails

As this makes web add-ins inconsistent across different platforms and does not provide the expected results in the classic Outlook for Windows, this should be fixed. This is preventing customers from migrating from VSTO-based add-ins to Web add-ins.

## Steps to reproduce

1. Set up a custom template applied to new documents (update normalemail.dotm file). Can be easily tested with line spacing set to e.g. 40pt.
2. Use Script Lab for Outlook
3. Insert a signature using the "Work with client signatures (compose)" sample. The signature will inherit the styles. If custom styles are provided using HTML, then those will be overwritten when inserted.

**Comment by bogdanst24** [comment]
Hi @pinverso, any reason why the tags were removed?

**Comment by pinverso** [comment]
Hello @bogdanst24 , I added back the Needs:  attention.   It was inadvertently removed.

**Comment by ztcdsb** [comment]
@bogdanst24 
Thanks for reporting this issue regarding signature. It has been put on our backlog. We unfortunately have no timelines to share at this point 

 Internal tracking id: Office: 9445392

**Comment by ndeleuze** [comment]
@bogdanst24 I am having trouble reproducing this issue, as I see that Win32 and Outlook Online both ignore the custom formatting applied to the email. Would you be able to provide an example normalemail.dotm and call to setSignatureAsync with HTML? Thank you.

**Comment by bogdanst24** [comment]
I can't upload here a .dotm file, so I uploaded it on my git and added you as a collaborator

**Comment by bogdanst24** [comment]
@ndeleuze have you managed to get access to it?

**Comment by ndeleuze** [comment]
@bogdanst24 sorry for the late reply. This invitation has expired. Would you be able to resend it?

**Comment by bogdanst24** [comment]
Just did it

**Comment by ndeleuze** [comment]
@bogdanst24 thank you, I have access. However, after replacing the NormalEmail.dotm file in `AppData\Roaming\Microsoft\Templates` with the file you provided, I don't see any style changes when opening a new email. It seems that the template you provided should have me using "Agency FB" font as a default when opening new compose emails, but I don't see this.

Is this how you went about updated Outlooks default compose template?

Would you also be able to provide the exact example call to setSignatureAsync with HTML?

I want to make sure that my repro is the same as yours.

Thanks.

**Comment by ndeleuze** [comment]
@bogdanst24 I am able to alter the default Outlook style. On my end I am seeing that the signature inserted (using the ScriptLab sample) seems to honor the new style: the signature is inserted with 42 pt line spacing like the rest of the document. I am seeing the same behavior for native signatures. However, Outlook on the web does not seem to honor this.

I this accurate to what you are seeing?

**Comment by bogdanst24** [comment]
Yes, it is accurate. And the issue is as reported only for Outlook for Windows - the classic/old one

**Comment by ndeleuze** [comment]
@bogdanst24, do you know how many customers are affected by this?

**Comment by bogdanst24** [comment]
We got bug reports from 4 customers, but we don't know how many are affected

**Comment by ndeleuze** [comment]
@bogdanst24 apologies for the late reply. I would like to understand your desired behavior, as you mentioned that this issue relates only to classic Outlook for Windows.

The behavior that I see is as follows:
1. Win32 classic Outlook: Native signatures and web Add-in signatures behave the same (both **honor** the normalemail.dotm file).
2. Outlook on the web: Native signatures and web Add-in signatures behave the same (both **ignore** the default email format set on the settings page).

Please let me know if your experience differs from mine. But with this in mind, what is the desired behavior? It sounds like you would prefer different behavior from Win32 classic. Is that correct?

**Comment by bogdanst24** [comment]
It is definitely expected that the signatures are not influenced by normalemail.dotm files. And the reasons:
- It is the New Outlook's behavior, which kind of shows this is how Microsoft also things it should be, as it was build like that
- classic Outlook provides a "signature designer", which is completely useless if the designed signature is either way changed at insertion

**Comment by ndeleuze** [comment]
@bogdanst24, apologies for the delay in response. After careful consideration, our team has decided to not fix this issue in the short term. We recommend you to upvote or create a new [Tech Community](http://aka.ms/M365dev-suggestions) item, this will give us visibility of the commonality of it and will help with our prioritization. If we end up working on this issue, we will re-activate as needed.
